sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuTransitionOverrides.scala

```diff
@@ -407,7 +407,12 @@ class GpuTransitionOverrides extends Rule[SparkPlan] {
     }
   }
 
-  private def insertHashOptimizeSorts(plan: SparkPlan): SparkPlan = {
+  // Find  GpuDataWritingCommandExec(......(GpuHashJoin)),
+  //       GpuDataWritingCommandExec(......(GpuHashAggregateExec))
+  // from top to bottom and insert sort to optimize the file size.
+  // such as: GpuDataWritingCommandExec(GpuProject(GpuHashAggregateExec))
+  private def insertHashOptimizeSorts(plan: SparkPlan,
+                                      hasWriteParent:Boolean = false): SparkPlan = {
```

Comment:
nit: indent doesn't match style guide https://github.com/databricks/scala-style-guide#indent

```suggestion
                                      hasWriteParent:Boolean = false): SparkPlan = {
      hasWriteParent:Boolean = false): SparkPlan = {
```